deltacat/storage/iceberg/model.py

```diff
@@ -502,10 +503,9 @@ def unmap(
                 final_fields.append(final_field)
             iceberg_schema = IcebergSchema(
                 fields=final_fields,
-                schema_id=obj.linked_schema_ids.get(stream_locator, INITIAL_SCHEMA_ID)
-                if obj.linked_schema_ids is not None
-                else INITIAL_SCHEMA_ID,
-                identifier_field_ids=obj.merge_keys,
```

Comment:
this is a hack I did to make my POC work, I don't think this is a proper fix though, I'm also not sure what a proper fix should look like due to lack of context.

I'm fine keeping those so at least we have some unit test working comparing to simply breaking, but we should create an issue or TODO for tracking.